input/pagecontent/reference.md

```diff
@@ -3010,249 +3010,10 @@ Following [standard profiling rules established in FHIR](https://www.hl7.org/fhi
-
-                 //        YEAR         ( -   MONTH   ( -    DAY    ( T TIME )?)?)?
-DATETIME:           [0-9][0-9][0-9][0-9]('-'[0-9][0-9]('-'[0-9][0-9]('T' TIME)?)?)?;
-
-                 //    HOUR   ( :   MINUTE  ( :   SECOND  ( . MILLI )?)?)?( Z  |     +/-        HOUR   :  MINUTE   )?
-TIME:               [0-9][0-9](':'[0-9][0-9](':'[0-9][0-9]('.'[0-9]+)?)?)?('Z' | ('+' | '-')[0-9][0-9]':'[0-9][0-9])?;
-
-                 // DIGITS  ..  (DIGITS |  * )
-CARD:               ([0-9]+)? '..' ([0-9]+ | '*')?;
-
-              //  Reference       (        ITEM         |         ITEM         )
-REFERENCE:       'Reference' WS* '(' WS* SEQUENCE WS* (WS 'or' WS+ SEQUENCE WS*)* ')';
-
-                 // Canonical       (              URL|VERSION               or              URL|VERSION             )
-CANONICAL     :    'Canonical' WS* '(' WS* SEQUENCE ('|' SEQUENCE)? WS* (WS 'or' WS+ SEQUENCE ('|' SEQUENCE)? WS*)* ')';
-
-                 //  ^  NON-WHITESPACE
-CARET_SEQUENCE:     '^' NONWS+;
-
-                 // '/' EXPRESSION '/'
-REGEX:              '/' ('\\/' | ~[*/\r\n])('\\/' | ~[/\r\n])* '/';
-
-PARAMETER_DEF_LIST: '(' (SEQUENCE WS* COMMA WS*)* SEQUENCE ')';
-
-// BLOCK_COMMENT must precede SEQUENCE so that a block comment without whitespace does not become a SEQUENCE
-BLOCK_COMMENT:      '/*' .*? '*/' -> skip;
-                 // NON-WHITESPACE
-SEQUENCE:           NONWS+;
-
-
-
-// FRAGMENTS
-fragment WS: [ \t\r\n\f\u00A0];
-fragment NONWS: ~[ \t\r\n\f\u00A0];
-fragment NONWS_STR: ~[ \t\r\n\f\u00A0\\"];
-
-// IGNORED TOKENS
-WHITESPACE:         WS -> channel(HIDDEN);
-LINE_COMMENT:       '//' ~[\r\n]* [\r\n] -> skip;
-
-mode RULESET_OR_INSERT;
-PARAM_RULESET_REFERENCE:      WS* NONWS+ (WS* ('(' ('\\)' | '\\\\' | ~[)])+ ')')) -> popMode;
-RULESET_REFERENCE:            WS* NONWS+ -> popMode;
-```
+[SUSHI](https://github.com/FHIR/sushi) provides an implementation of a FSH language parser described in [ANTLR v4](https://www.antlr.org/). It includes elements of the FSH language marked as {%include tu.html%}. The entity names defined in the grammar may not correspond to those used in the language specification. If there is a conflict between the language specification and the grammar defined in this Appendix, the language specification takes precedence. This grammar implementation is provided for informational purposes and is not normative.
+
+The latest version of the parser grammar can be found [here](https://github.com/FHIR/sushi/blob/v3.0.0-beta.2/antlr/src/main/antlr/FSH.g4).
+
+The latest version of the lexer grammar can be found [here](https://github.com/FHIR/sushi/blob/v3.0.0-beta.2/antlr/src/main/antlr/FSHLexer.g4).
+
+Both are implemented in SUSHI `v3.0.0-beta.2` and included as part of the released version of SUSHI, which is available through [npm](https://www.npmjs.com/package/fsh-sushi/v/3.0.0-beta.2) and on [GitHub](https://github.com/FHIR/sushi/releases/tag/v3.0.0-beta.2).
```

Comment:
I don't know if this last paragraph is necessary. It concerns me a little because we need to make it link to a specific version of SUSHI (which will quickly be outdated) or link to the main branch (which may contain _additional_ grammar changes). I think we may be better off not linking to SUSHI (as an app) here at all. I think the links to the grammars are sufficient (and _should_ be linked to a specific tag/branch, as done here).

Reply:
I think that's a good point. I removed the last paragraph in [62d5a11](https://github.com/HL7/fhir-shorthand/pull/147/commits/62d5a115697ebc22913da0c4dd8075faf0b2d24e).